cfgov/preprocessed/css/organisms/sidebar-breakout.less

```diff
@@ -0,0 +1,152 @@
+/* topdoc
+  name: Sidebar Breakout
+  family: cfgov-organisms
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="o-sidebar-breakout
+                    content__flush-bottom">
+            <div class="content_intro">
+                <div class="content-l">
+                    <section class="o-sidebar-breakout_col
+                                    content-l_col
+                                    content-l_col-2-3">
+                        <m-text-introduction></m-text-introduction>
+                    </section>
+                    <aside class="o-sidebar-breakout_col
+                                  content-l_col
+                                  content-l_col-1-3
+                                  block
+                                  block__flush-top
+                                  block__flush-bottom">
+                        <h3 class="o-sidebar-breakout_heading">
+                        </h3>
+                        <div class="o-sidebar-breakout_img-container
+                                    u-centered-on-mobile">
+                            <div class="o-sidebar-breakout_img
+                                        u-centered-on-mobile"
+                                 style="background-image:url()">
+                            </div>
+                        </div>
+                        <div class="o-sidebar-breakout_text-container">
+                            <h4 class="o-sidebar-breakout_text-heading">
+                                <span class="cf-icon"></span>
+                            </h4>
+                            <p class="o-sidebar-breakout_text-body short-desc">
+                            </p>
+                            <m-related-post></m-related-posts>
+                        </div>
+                    </aside>
+                </div>
+            </div>
+        s</div>
+        <div class="content_line"></div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-sidebar-breakout
+            .content_intro
+              .content-l
+                .o-sidebar-breakout_col
+                    .m-text-introduction
+                .o-sidebar-breakout_col
+                  .o-sidebar-breakout_heading
+                  .o-sidebar-breakout_img-container
+                    .o-sidebar-breakout_img
+                  .o-sidebar-breakout_text-container
+                    .o-sidebar-breakout_text-heading
+                      .cf-icon
+                    .o-sidebar-breakout_text-body
+                    .m-related-posts
+          .content_line
+  tags:
+    - cfgov-organisms
+*/
+
+@margin__em: unit( @grid_gutter-width / @base-font-size-px, em );
+@image_width__px:  300px;
+@image_height__px: 170px;
```

Comment:
Probably best to prefix these with the organism name to avoid colliding with other organisms that set image size and follow this precedent. E.g. https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/preprocessed/css/molecules/image-text-25-75.less#L16-L17 Or if there was global consistency we could pull these image sizes out of all organisms/molecules and set global values like the breakpoints.